### PR TITLE
Highlight styles

### PR DESCRIPTION
**Pull request by stacyk**
## Description
Edit highlight styles

- make highlight colors variables (especially if used more than once)
- edited enhanced a color or two in order to differentiate it from the other similar reds colors (mostly the magenta ones).
- added the 600 weight (and italics) so the browser doesn't force a weight/style that didn't exist (most noticeable in Safari)

## Steps to test/reproduce
- review code samples on the documentation pages (not JS pages) ie. This is a good page to see some of the changes https://deploy-preview-56--sass-site-oddbird.netlify.app/documentation/interpolation/#in-sass-script

## Show me

Note: The items that are highlighted in a completely different color is a result of the library we are using now which doesn't wrap quite as much in spans like the previous library did.
after-left-before-right

### CODE REVIEW

**Comment by stacyk** [comment]
[Syntax Highlighting Style cleanup](https://trello.com/c/OgN5xbo3/68-syntax-highlighting-style-cleanup)

**Comment by jgerigmeyer** [comment]
@stacyk I assume this is still in-progress/draft at this point?

**Review by mirisuzanne — APPROVED** [review]
looks great!

File: source/assets/sass/visual-design/_theme.scss [hunk 1]
```diff
@@ -3,9 +3,12 @@
...
 html {
-  @include config.tokens(config.$scale);
-  @include config.tokens(config.$brand-colors);
-  @include config.tokens(config.$content-colors);
+  &,
+  &::selection {
+    @include config.tokens(config.$scale);
+    @include config.tokens(config.$brand-colors);
+    @include config.tokens(config.$content-colors);
+  }
```
Comment (stacyk) [inline source/assets/sass/visual-design/_theme.scss:12]: @mirisuzanne this is how I organized it and this is working locally.

Comment (mirisuzanne) [reply source/assets/sass/visual-design/_theme.scss:12]: looks great

**Review by mirisuzanne — APPROVED** [review]

**Pull request merged**